### PR TITLE
Extract baserom calls from various battle data files and crt0.s

### PR DESCRIPTION
**Pull request by Diegoisawesome**

### CODE REVIEW

File: data/battle_4.s [hunk 1]
```diff
@@ -7,44 +7,134 @@
...
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB36A
+	.4byte gUnknown_082DB3F4
+
+	.align 2
+	window_template 0x00, 0x01, 0x03, 0x07, 0x0f, 0x1f, 0x003f
```
Comment (ProjectRevoTPP) [inline data/battle_4.s]: is this really a window template?

File: data/battle_4.s [hunk 2]
```diff
@@ -7,44 +7,134 @@
...
+	.align 2
+gUnknown_0831C3C0:: @ 831C3C0
+	.2byte 0xd75a @ tileTag
+	.2byte 0xd75a @ paletteTag
+	.4byte gUnknown_0831C3B8
+	.4byte gDummySpriteAnimTable
+	.4byte NULL
+	.4byte gDummySpriteAffineAnimTable
+	.4byte sub_804F450
+
```
Comment (ProjectRevoTPP) [inline data/battle_4.s]: define a macro for this?

File: data/battle_4.s [hunk 3]
```diff
@@ -7,44 +7,134 @@
...
+	.align 2
 sProtectSuccessRates:: @ 831C3D8
-	.incbin "baserom.gba", 0x31c3d8, 0x8
+	.2byte 0xffff, 0x7fff, 0x3fff, 0x1fff
 
+	.align 2
 gUnknown_0831C3E0:: @ 831C3E0
-	.incbin "baserom.gba", 0x31c3e0, 0x28
+	.2byte 0x0076, 0x00a5, 0x00a6, 0x0066
```
Comment (ProjectRevoTPP) [inline data/battle_4.s]: move list?

File: data/battle_4.s (hunk 1 above)
Comment (Diegoisawesome) [reply data/battle_4.s]: Not 100% sure but it fits perfectly with the pattern I've consistently seen for all window templates.

File: data/battle_4.s (hunk 2 above)
Comment (Diegoisawesome) [reply data/battle_4.s]: Is the intermediate step of a macro absolutely necessary? It's going to end up in a C struct anyway.

**Pull request merged**